Form-Controls/index.html

```diff
@@ -10,18 +10,38 @@
 </head>
 <body>
   <header>
-    <h1>Product Pick</h1>
+    <h1>T-shirt Order Form</h1>
   </header>
   <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
+  <form id="order-form">
+    <!-- Form fields must include a validated name, validated email address, 3 colour options only, 6 sizes only and delivery date -->
+    <!--Customer name-->
+    <div class="form-contents">
+        <label for="Customer name">Your name:</label>
+        <input type="text" id="Customer name" name="Customer name" minlength="2" required placeholder="Please enter your name">
+    </div>
+
+    <!-- Customer email-->
+    <div class="form-contents">
+        <label for="Customer email">Your email address: </label>
+        <input type="email" id="Email address" name="Customer email" required placeholder="Please enter your email address">
+    </div>
+  
+  <!--Color selection-->
+    <div class="form-contents">
+        <label for="T-shirt color">Select your t-shirt colour:</label>
+        <select id="T-shirt color" name="T-shirt color" required>
+        <option value="" disabled selected>Select your colour</option>  
+        </select>
+    </div>
+  
+  
```

Comment:
Please add a submit button.

Reply:
![Screenshot from 2024-10-25 12-37-52](https://github.com/user-attachments/assets/81cb0e15-7430-4172-80b1-de8289b84fef)
Hi Waldo. Thank you so much. I have resolved the HTML that was displaying, added the submit button and only left one H1 heading.